picard/ui/ui_options_interface.py

```diff
@@ -117,6 +120,7 @@ def retranslateUi(self, InterfaceOptionsPage):
         self.builtin_search.setText(_("Use builtin search rather than looking in browser"))
         self.use_adv_search_syntax.setText(_("Use advanced query syntax"))
         self.quit_confirmation.setText(_("Show a quit confirmation dialog for unsaved changes"))
+        self.file_save_warning.setText(_("Show a confirmation dialog when saving files"))
```

Comment:
I know it may seem a little odd to provide an option to reset the first use warning, but I am personally reasonably confident that at some point someone is going to regret clearing this flag and want to reset it.

Reply:
I'm not sure that's still the case with the shortened version of the new user dialog?

Reply:
I'd still allow bringing this dialog back if wanted.